docs_versioned_docs/version-ros2humble/ros/config/yaml/platform.mdx

```diff
@@ -50,280 +53,86 @@ For systems without a BMS, the state of charge should be treated as a rough esti
 :::
 
 
-<details><summary>A200</summary>
-<p>
-
-<table>
-  <thead>
-    <tr>
-      <th>Battery</th>
-      <th>Chemistry</th>
-      <th>Nominal Voltage (V)</th>
-      <th>Capacity (Ah)</th>
-      <th>Configurations</th>
-    </tr>
-  </thead>
-  <tbody>
-    <tr>
-      <td><code>ES20_12C</code></td>
-      <td>Lead Acid</td>
-      <td>12.0</td>
-      <td>20.0</td>
-      <td>S2P1</td>
-    </tr>
-    <tr>
-      <td><code>HE2613</code></td>
-      <td>Lithium Ion</td>
-      <td>25.9</td>
-      <td>12.8</td>
-      <td>S1P3<br/>S1P4</td>
-    </tr>
-  </tbody>
-</table>
-
-</p>
-</details>
+<Tabs groupId="platform">
+  <TabItem value="A200" label="A200" default>
 
-<details><summary>J100</summary>
-<p>
+| Battery | Chemistry | Nominal Voltage (V) | Capacity (Ah) | Configurations |
+|:-:|:-:|:-:|:-:|:-:|
+| `ES20_12C` | Lead Acid | 12.0 | 20.0 | S2P1 |
+| `HE2613` | Lithium Ion | 25.9 | 12.8 | S1P3<br/>S1P4 |
```

Comment:
@hilary-luo,  should _Lithium Ion_ change to one of these options?

- Lithium Iron Phosphate
- LiFePO₄
- LFP

Reply:
I was just reorganizing how this information is displayed (formatting of tables and tabs) so I am not sure if that is desired or what the correct answer would be. That isn't the only battery that we report as Lithium Ion. I can ask around though.

Reply:
@rhysfaultless-cpr which would you prefer?